apps/perf-test/src/scenarios/MakeStyles.tsx

```diff
@@ -6,20 +6,20 @@ const renderer = createDOMRenderer(document);
 const useStyles = makeStyles({
   view: {
     alignItems: 'stretch',
-    borderWidth: 0,
-    borderStyle: 'solid',
+    borderLeftWidth: 0,
```

Comment:
this could invalidate previous perf results, shouldn't we just use the css shorthand functions instead ?

Reply:
```suggestion
    borderWidth: ...shorthands.borderWidth(0),
```

Reply:
There are no difference (I had results from the main PR):
![image](https://user-images.githubusercontent.com/14183168/143850403-8b579b81-283e-4384-ac79-70d991af1e7f.png)

_Why?_ This test measures re-render price, not style rendering or insertion.

Reply:
Ah ok, yeah the different aspects of perf testing :)

Reply:
I thought that less rules inserted would make a difference but in rerender numbers rules are only inserted once anyways